tests/check/check_h26xsigned_auth.c

```diff
@@ -859,7 +863,36 @@ START_TEST(camera_reset_on_signing_side)
   // will be marked as invalid and compute 3 more NALUs than expected. In G it is communicated
   // there is only 2 NALUs present (GI). So missed NALUs equals -3 (IPP).
   const struct validation_stats expected = {
-      .valid_gops = 2, .invalid_gops = 2, .missed_nalus = -3, .pending_nalus = 4
+      .valid_gops = 2, .invalid_gops = 2, .missed_nalus = -3, .pending_nalus = 4, .public_key_has_changed = true
```

Comment:
Add a TODO above this declaration that public_key_has_changed is expected to be true now when we have changed the behavior in generate private key.